h_transport_materials/property_database/pdag/palladium_silver.py

```diff
@@ -0,0 +1,149 @@
+import h_transport_materials as htm
+from h_transport_materials import(
+    Diffusivity, 
+    Solubility, 
+    Permeability,
+    DissociationCoeff, 
+    RecombinationCoeff)
+from pathlib import Path
+import numpy as np
+
+serra_diffusivity_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/serra_1998/diffusivity.csv",
+    delimiter=",",
+    names=True,
+)
+
+serra_diffusivity_h = Diffusivity(
+    data_T=1000 / serra_diffusivity_data["hydrogenX"] * htm.ureg.K,
+    data_y=serra_diffusivity_data["hydrogenY"] * htm.ureg.m**2 * htm.ureg.s**-1,
+    isotope="H",
+    source="serra_hydrogen_1998-1",
+)
+
+serra_diffusivity_d = Diffusivity(
+    data_T=1000 / serra_diffusivity_data["deuteriumX"] * htm.ureg.K,
+    data_y=serra_diffusivity_data["deuteriumY"] * htm.ureg.m**2 * htm.ureg.s**-1,
+    isotope="D",
+    source="serra_hydrogen_1998-1",
+)
+
+serra_solubility_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/serra_1998/solubility.csv",
+    delimiter=",",
+    names=True,
+)
+
+serra_solubility_h = Solubility(
+    units="m-3 Pa-1/2",
+    data_T=1000 / serra_solubility_data["hydrogenX"] * htm.ureg.K,
+    data_y=serra_solubility_data["hydrogenY"]
+    * htm.ureg.mol
+    * htm.ureg.m**-3
+    * htm.ureg.Pa**-0.5,
+    isotope="H",
+    source="serra_hydrogen_1998-1",
+)
+
+serra_solubility_d = Solubility(
+    units="m-3 Pa-1/2",
+    data_T=1000 / serra_solubility_data["deuteriumX"] * htm.ureg.K,
+    data_y=serra_solubility_data["deuteriumY"]
+    * htm.ureg.mol
+    * htm.ureg.m**-3
+    * htm.ureg.Pa**-0.5,
+    isotope="D",
+    source="serra_hydrogen_1998-1",
+)
+serra_dissociation_h = DissociationCoeff(
+    pre_exp=1.7e-2
+    * htm.ureg.mol
+    * htm.ureg.m**-2
+    * htm.ureg.s**-1
+    * htm.ureg.Pa**-1,
+    act_energy=26.294 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="H",
+    range=(323 * htm.ureg.K, 773* htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+)
+serra_dissociation_d = DissociationCoeff(
+    pre_exp=1.3e-2
+    * htm.ureg.mol
+    * htm.ureg.m**-2
+    * htm.ureg.s**-1
+    * htm.ureg.Pa**-1,
+    act_energy=24.780 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="D",
+    range=(323 * htm.ureg.K, 773* htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+)
+serra_recombination_coeff_h= RecombinationCoeff(
+    pre_exp=0.51 * htm.ureg.mol**-1 * htm.ureg.m**4 * htm.ureg.s**-1,
+    act_energy=65.490 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="H",
+    range=(323 * htm.ureg.K, 773 * htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+) 
+serra_recombination_coeff_d= RecombinationCoeff(
+    pre_exp=0.39 * htm.ureg.mol**-1 * htm.ureg.m**4 * htm.ureg.s**-1,
+    act_energy=61.842 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="D",
+    range=(323 * htm.ureg.K, 773 * htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+) 
+vadrucci_permeability_wt84 = Permeability(
+    pre_exp=2.95e-8 * u.mol * u.m**-1 * u.s**-1 * u.Pa**-0.5,
+    act_energy=25.3149 * u.kJ * u.mol**-1,
```

Comment:
From what I see in Table 2 of the paper, the activation energy is 2.5 kJ/mol and not 25 kJ/mol.
To avoid this kind of typo, I try to always write the parameters with the same units as in the paper (here J/mol instead of kJ/mol).
```suggestion
    act_energy=2531.49 * htm.ureg.J * htm.ureg.mol**-1,
```